js-quantities/index.d.ts

```diff
@@ -1,79 +1,73 @@
-// Type definitions for JS-quantities
+// Type definitions for JS-quantities 1.6
 // Project: http://gentooboontoo.github.io/js-quantities/
-// Definitions by: William Comartin <https://github.com/wcomartin>
+// Definitions by: William Rummler <https://github.com/wrummler>
```

Comment:
@vvakame No need to apologize, I have no claim to your time. Many thanks to you and all the other maintainers of this project.

I'm not really comfortable making the requested revision, at least on the basis of whether there was any level of meaningful contribution in the original definitions. The original definitions were more than a little mismatched with this library's actual API, as can be verified by testing the original version of this file against the test file submitted in this PR. The effect of the originals' inaccuracy wasn't neutral, either. My team (new to TS and the modern JS ecosystem in general) lost some time at one point until realizing our problems were due to the non-trivial inaccuracies of these definitions.

If DT policy requires this revision anyway, please let me know and I'll make the necessary update. Thanks again.